From: KevinDuan <duankaiwen@kylinos.cn>
Date: Wed, 24 Jul 2024 03:04:20 +0000
Subject: Translated using Weblate (Kyrgyz)

Currently translated at 91.6% (22 of 24 strings)

Translation: openkylin-nile-new/kylin-file-crush
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-file-crush/ky/
---
 .../file-smash/translations/kylin-file-crush_kk.ts | 46 ++++++++++-----------
 .../file-smash/translations/kylin-file-crush_ky.ts | 48 +++++++++++-----------
 2 files changed, 47 insertions(+), 47 deletions(-)

diff --git a/treasure-chest/file-smash/translations/kylin-file-crush_kk.ts b/treasure-chest/file-smash/translations/kylin-file-crush_kk.ts
index 89e52cf..d15ea44 100644
--- a/treasure-chest/file-smash/translations/kylin-file-crush_kk.ts
+++ b/treasure-chest/file-smash/translations/kylin-file-crush_kk.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../alertdialog.ui" line="14"/>
         <source>Form</source>
-        <translation>Пішін</translation>
+        <translation>ٴپىشىنى</translation>
     </message>
 </context>
 <context>
@@ -14,12 +14,12 @@
     <message>
         <location filename="../kalertdialog.cpp" line="47"/>
         <source>The file you selected is a directory or does not have permissions, cannot be shredded!</source>
-        <translation>Таңдалған файл каталог болып табылады немесе рұқсаттары жоқ, ұсақталуы мүмкін емес!</translation>
+        <translation>ٴسىز تالداعان حۇجات باس مازمۇن ياكي ۇقىعى جوق، پارشالانعان بولمايدى!</translation>
     </message>
     <message>
         <location filename="../kalertdialog.cpp" line="50"/>
         <source>sure</source>
-        <translation>сенімді</translation>
+        <translation>تۇراقتاندىرۋ</translation>
     </message>
 </context>
 <context>
@@ -27,22 +27,22 @@
     <message>
         <location filename="../ksuredialog.cpp" line="45"/>
         <source>Are you sure to start crushing?</source>
-        <translation>Сіз міндетті түрде ұсақтауды бастайсыз ба?</translation>
+        <translation>پارشالاۋدى باستايسٸزبا؟</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="51"/>
         <source>Crushed files cannot be recovered!</source>
-        <translation>Бөлшектелген файлдарды қалпына келтіру мүмкін емес!</translation>
+        <translation>پارشالانعان حۇجاتتاردى قالپىنا كەلتىرگەلى بولمايدى!</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="58"/>
         <source>sure</source>
-        <translation>сенімді</translation>
+        <translation>تۇراقتاندىرۋ</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="66"/>
         <source>cancle</source>
-        <translation>канкл</translation>
+        <translation>بوس ەتۋ</translation>
     </message>
 </context>
 <context>
@@ -50,7 +50,7 @@
     <message>
         <location filename="../main.cpp" line="77"/>
         <source>kylin file crush</source>
-        <translation>Ұсақталған менеджер</translation>
+        <translation></translation>
     </message>
 </context>
 <context>
@@ -63,33 +63,33 @@
         <location filename="../shreddialog.cpp" line="35"/>
         <location filename="../shreddialog.cpp" line="38"/>
         <source>File Shredder</source>
-        <translation type="unfinished">Ұсақталған менеджер</translation>
+        <translation></translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="60"/>
         <source>No file selected to be shredded</source>
-        <translation>Ұсақталу үшін файл таңдалмаған</translation>
+        <translation>پارشالايتىن حۇجات تالدانبادٸ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="159"/>
         <location filename="../shreddialog.cpp" line="204"/>
         <source>The file is completely shredded and cannot be recovered</source>
-        <translation>Файл толығымен ұсақталады және қалпына келтірілмеді</translation>
+        <translation>حۇجات بۇتٸندەپ پارشالانادى، ۋا قالپىنا كەلتىرگەلى بولمايدى</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="160"/>
         <source>Shred File</source>
-        <translation>Ұсақталған файл</translation>
+        <translation>پارشالاۋ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="161"/>
         <source>Deselect</source>
-        <translation>Жоққа шығару</translation>
+        <translation>جوتكەۋ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="162"/>
         <source>Add File</source>
-        <translation>Файлды қосу</translation>
+        <translation>قوسۋ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="163"/>
@@ -99,37 +99,37 @@
     <message>
         <location filename="../shreddialog.cpp" line="195"/>
         <source>Select file</source>
-        <translation>Файлды таңдау</translation>
+        <translation>حۇجات تالداۋ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="195"/>
         <source>All Files(*)</source>
-        <translation>Барлық файлдар(*)</translation>
+        <translation>بارلٸق حۇجات(*)</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="236"/>
         <source>File Shredding ...</source>
-        <translation>Файлды ұсақтау ...</translation>
+        <translation>حۇجات پارچىلىنىۋاتىدۇ...</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="259"/>
         <source>Shred successfully!</source>
-        <translation>Ойдағыдай ұсақтап қағып ал!</translation>
+        <translation>پارشالاۋ جەڭىسكە جەتۋ قالدى!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="273"/>
         <source>Shred failed!</source>
-        <translation>Ұсақталған сәтсіздікке ұшырады!</translation>
+        <translation>پارشالاۋ جەڭىلىپ قالدى!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="295"/>
         <source>Please select the file to shred!</source>
-        <translation>Файлды ұсақтап таңдауыңызды сұраймын!</translation>
+        <translation>پارشالايتىن حۇجاتتى تالدا!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="300"/>
         <source>You did not select a file with permissions!</source>
-        <translation>Рұқсаттары бар файлды таңдаған жоқсыз!</translation>
+        <translation>تاللغىنىڭىز ۇقىعى جوق حۇجات!</translation>
     </message>
 </context>
 <context>
@@ -137,12 +137,12 @@
     <message>
         <location filename="../shredmanager.cpp" line="49"/>
         <source>Shred Manager</source>
-        <translation>Ұсақталған менеджер</translation>
+        <translation>حۇجات پارشالاعىش</translation>
     </message>
     <message>
         <location filename="../shredmanager.cpp" line="54"/>
         <source>Delete files makes it unable to recover</source>
-        <translation>Файлдарды жою қалпына келтіруге мүмкіндік бермейді</translation>
+        <translation>پارشالانعان حۇجاتتى قالپىنا كەلتىرگەلى بولمايدى!</translation>
     </message>
 </context>
 </TS>
diff --git a/treasure-chest/file-smash/translations/kylin-file-crush_ky.ts b/treasure-chest/file-smash/translations/kylin-file-crush_ky.ts
index 2ca7d46..1f9ffee 100644
--- a/treasure-chest/file-smash/translations/kylin-file-crush_ky.ts
+++ b/treasure-chest/file-smash/translations/kylin-file-crush_ky.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../alertdialog.ui" line="14"/>
         <source>Form</source>
-        <translation>Форма</translation>
+        <translation>فورماتى</translation>
     </message>
 </context>
 <context>
@@ -14,12 +14,12 @@
     <message>
         <location filename="../kalertdialog.cpp" line="47"/>
         <source>The file you selected is a directory or does not have permissions, cannot be shredded!</source>
-        <translation>Сиз тандап алган файл каталог болуп саналат же уруксаты жок, майдаланбайт!</translation>
+        <translation>سىز تاندالعان  ۅجۅت تىزىمدىك كۅرۉنۉشتۅرۉ ۇقۇعۇ جوق، پارچالووعو بولبويت !</translation>
     </message>
     <message>
         <location filename="../kalertdialog.cpp" line="50"/>
         <source>sure</source>
-        <translation>албетте</translation>
+        <translation>بەكىتۉۉ</translation>
     </message>
 </context>
 <context>
@@ -27,22 +27,22 @@
     <message>
         <location filename="../ksuredialog.cpp" line="45"/>
         <source>Are you sure to start crushing?</source>
-        <translation>Сиз майдалап баштай турганы шексизби?</translation>
+        <translation>پارچىلاشنى باشتايسىزبى؟</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="51"/>
         <source>Crushed files cannot be recovered!</source>
-        <translation>Майдаланган файлдарды калыбына келтирүү мүмкүн эмес!</translation>
+        <translation>پارچىلانغان  ۅجۅتتۅرۉن العاچىنا كەلتىرگەلى بولبويت !</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="58"/>
         <source>sure</source>
-        <translation>албетте</translation>
+        <translation>بەكىتۉۉ</translation>
     </message>
     <message>
         <location filename="../ksuredialog.cpp" line="66"/>
         <source>cancle</source>
-        <translation>канкл</translation>
+        <translation>بەكەر  جاسوو ،اتقارۇۇ</translation>
     </message>
 </context>
 <context>
@@ -50,7 +50,7 @@
     <message>
         <location filename="../main.cpp" line="77"/>
         <source>kylin file crush</source>
-        <translation>килин файлын талкалоо</translation>
+        <translation></translation>
     </message>
 </context>
 <context>
@@ -63,73 +63,73 @@
         <location filename="../shreddialog.cpp" line="35"/>
         <location filename="../shreddialog.cpp" line="38"/>
         <source>File Shredder</source>
-        <translation>Файл Шреддер</translation>
+        <translation></translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="60"/>
         <source>No file selected to be shredded</source>
-        <translation>Майдалоо үчүн тандалган файл жок</translation>
+        <translation>پارچىلايدىغان ۅجۅت تاندالبادى</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="159"/>
         <location filename="../shreddialog.cpp" line="204"/>
         <source>The file is completely shredded and cannot be recovered</source>
-        <translation>Файл толугу менен майдаланган жана калыбына келтирилбейт</translation>
+        <translation>ۅجۅت پۉتكۉلدۅي پارچىلىنىدۇ، جانا العاچىنا كەلتىرگەلى بولبويت</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="160"/>
         <source>Shred File</source>
-        <translation>Майдаланган файл</translation>
+        <translation>مايدالاش</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="161"/>
         <source>Deselect</source>
-        <translation>Деселект</translation>
+        <translation>جۅتكۅۅ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="162"/>
         <source>Add File</source>
-        <translation>Файлды кошуу</translation>
+        <translation>قوشۇۇ</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="163"/>
         <source>Note:File shredding cannot be cancelled, please exercise caution!</source>
-        <translation>Эскертүү:Файлды майдалоону жокко чыгаруу мүмкүн эмес, сак болгула!</translation>
+        <translation>دىققات:  ۅجۅتۉن مايدالاش  ماشعۇلاتىنى بەكەر  جاساعالى بولبويت ، ەتىيات مەنەن ماشقۇلدانۇۇ  قىلىڭ!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="195"/>
         <source>Select file</source>
-        <translation>Файлды тандоо</translation>
+        <translation>ۅجۅت تانداش</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="195"/>
         <source>All Files(*)</source>
-        <translation>Бардык файлдар (*)</translation>
+        <translation>باردىق ۅجۅت(*)</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="236"/>
         <source>File Shredding ...</source>
-        <translation>Файл майдалоо ...</translation>
+        <translation>ۅجۅت پارچىلىنىۋاتىدۇ...</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="259"/>
         <source>Shred successfully!</source>
-        <translation>Майдаланган ийгиликтүү!</translation>
+        <translation>مايدالاش  جەڭىشتۉۉ بولدۇ !</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="273"/>
         <source>Shred failed!</source>
-        <translation>Майдаланган ишке ашпады!</translation>
+        <translation>مايدالاش  جەڭىلۉۉ بولدۇ !</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="295"/>
         <source>Please select the file to shred!</source>
-        <translation>Файлды майдалоо үчүн тандаңыз!</translation>
+        <translation>پارچىلايدىغان  ۅجۅتۉن تانداڭ!</translation>
     </message>
     <message>
         <location filename="../shreddialog.cpp" line="300"/>
         <source>You did not select a file with permissions!</source>
-        <translation>Сиз уруксаты бар файлды тандаган жоксуңар!</translation>
+        <translation>تاللغىنىڭىز ۇقۇعۇ جوق ۅجۅت!</translation>
     </message>
 </context>
 <context>
@@ -137,12 +137,12 @@
     <message>
         <location filename="../shredmanager.cpp" line="49"/>
         <source>Shred Manager</source>
-        <translation>Шред менеджери</translation>
+        <translation>ۅجۅت پارچىلىغۇچ</translation>
     </message>
     <message>
         <location filename="../shredmanager.cpp" line="54"/>
         <source>Delete files makes it unable to recover</source>
-        <translation>Файлдарды жоготуу аны калыбына келтире албайт</translation>
+        <translation>پارچىلانغان  ۅجۅتۉن العاچىنا كەلتىرگەلى بولبويت !</translation>
     </message>
 </context>
 </TS>
